esmvalcore/_main.py

```diff
@@ -3,17 +3,20 @@
 http://www.esmvaltool.org
 
 CORE DEVELOPMENT TEAM AND CONTACTS:
-  Veronika Eyring (PI; DLR, Germany - veronika.eyring@dlr.de)
-  Bouwe Andela (NLESC, Netherlands - b.andela@esciencecenter.nl)
-  Bjoern Broetz (DLR, Germany - bjoern.broetz@dlr.de)
+  Birgit Hassler (Co-PI; DLR, Germany - birgit.hassler@dlr.de)
+  Alistair Sellar (Co-PI; Met Office, UK - alistair.sellar@metoffice.gov.uk)
+  Bouwe Andela (NLeSC, Netherlands - b.andela@esciencecenter.nl)
```

Comment:
```suggestion
  Bouwe Andela (Netherlands eScience Center, Netherlands - b.andela@esciencecenter.nl)
```
I think we're not supposed to use that abbreviation